### PR TITLE
Sort out autoplay data

### PR DESCRIPTION
**Pull request by queengooborg**
This PR sorts out the data regarding the `autoplay` attribute on `<audio>` and `<video>` elements, which resolves https://github.com/mdn/browser-compat-data/issues/7342.

When looking at the data, I happened to notice that the only mention of autoplay prohibited with an audio track was a note on Safari iOS, but from my own development experience, I found that Chrome Desktop also prohibited autoplaying audio.  As such, I decided to add a new subfeature for the attribute on the `<video>` element according to my findings.

Additionally, I had marked the attribute on `<audio>` elements as unsupported for when the browsers blocked automatic playback of audio.  Although the browser may recognize the attribute and it may still be present on the interface counterpart, setting the attribute will do nothing.

### CODE REVIEW

File: html/elements/audio.json [hunk 1]
```diff
@@ -57,36 +57,40 @@
...
+                "version_removed": "53"
               },
               "safari": {
-                "version_added": "3.1"
+                "version_added": "3.1",
+                "version_removed": "17"
               },
               "safari_ios": {
-                "version_added": "3"
+                "version_added": false
```
Comment (caugner) [inline html/elements/audio.json:90]: Are you sure Safari iOS never supported this?

**Review by caugner — COMMENTED** [review]
Can we validate these with the collector?

Idea: Add an element with `autoplay`, and listen for the "play" event with a timeout.

File: html/elements/audio.json (hunk 1 above)
Comment (queengooborg) [reply html/elements/audio.json:90]: I am -- I verified this via manual testing and confirmed that audio never autoplayed on iOS!

**Comment by queengooborg** [comment]
Right now, the collector test is rudimentary and just looks for the IDL attribute.  We can certainly rewrite the tests, however!

**Comment by caugner** [comment]
> Right now, the collector test is rudimentary and just looks for the IDL attribute. We can certainly rewrite the tests, however!

@queengooborg Can you point me to where that test would need to be written?

**Pull request merged**